### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by AdedayoOpeyemi**
This pull request is to add the Calculator UI component.

Requirement for this milestone

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
-  In Calculator.js, create a React component that matches the provided design
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
```
Comment (Davidosky007) [inline src/App.js]: * Kindly use class-based components as it is a requirement for this milestone [Refer to the requirement here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @AdedayoOpeyemi 👋, 

Good job so far! 💪 
* Good looking Readme file ✔️ 
* Professional Pull request ✔️ 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 2]
```diff
@@ -4,40 +4,57 @@
 
 > One paragraph statement about the project.
```
Comment (oxenprogrammer) [inline README.md:5]: **Optional Changes**
- [ ] Let us try and replace this placeholder with a description of what our project is about.

**Review by oxenprogrammer — APPROVED** [review]
Hi **@AdedayoOpeyemi**, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**